### ISSUE TITLE
Feature request: GUI size and font size

### ISSUE THREAD

**Issue by privat** [TheThirdOne/rars#189]
There is no way to change the GUI size globally (only the font size of editor area is updatable).
This cause issue with high resolution screens or with screen projecting (UI is too small).

**Comment by ScanuNicco** [comment]
I agree that this would be a nice feature, and I wanted to share a workaround for linux. Running rars with the following command will double the UI size:
`java -Dsun.java2d.uiScale=2.0 -jar rars1_6.jar`
